ArduPlane/tiltrotor.cpp

```diff
@@ -68,7 +68,7 @@ void QuadPlane::tiltrotor_continuous_update(void)
         max_change = tilt_max_change(false);
         
         float new_throttle = constrain_float(SRV_Channels::get_output_scaled(SRV_Channel::k_throttle)*0.01, 0, 1);
-        if (tilt.current_tilt < 1) {
+        if (tilt_get_biased_tilt() < 1) {
```

Comment:
not sure if we should change this one, seems its for forward flight only.

Reply:
with the bias applied in tilt_slew() the transition to FW will be start from the biased tilt angle and the motors will be fully forward when tilt_get_biased_tilt==1 and current_tilt is (1-bias) (although current_tilt will continue to increase till it hits 1, and biased_tilt is constrained to [0-1]
So I think this is correct.

Reply:
ah, i see what you mean, there would be a step as the bias is removed.